cg/meta/compress/compress.py

```diff
@@ -193,6 +195,15 @@ def clean_fastq(self, sample_id: str, archive_location: str) -> bool:
             )
         return all_cleaned
 
+    def _can_fastqs_be_removed(self, compression_data: CompressionData) -> bool:
+        is_fastq_compression_done: bool = self.crunchy_api.is_fastq_compression_done(
+            compression_data
+        )
+        spring_file: File | None = self.hk_api.get_file_insensitive_path(
+            compression_data.spring_path
+        )
+        return (is_fastq_compression_done) or (spring_file and spring_file.is_archived)
```

Comment:
can this be a boolean variable called is_spring_file_archived? Just for some clarity in this chaos

Reply:
Done! Also managed to get the `is_archived` property wrong so there has been some changes already 😅